platforms/hyperledger-indy/charts/indy-node/templates/statefulset.yaml

```diff
@@ -150,7 +150,7 @@ spec:
             - name: {{ $.Values.node.name }}-config
               mountPath: /etc/indy/indy_config.py
               subPath: indy_config.py
-            - name: "{{ $.Values.configmap.domainGenesis }}"
+            - name: {{ $.Values.node.name }}-domain-transactions-genesis
```

Comment:
all nodes within one org use the same config map

Reply:
using organization name now in separate chart